awx/main/tests/unit/test_validators.py

```diff
@@ -12,12 +12,43 @@
     TEST_OPENSSH_KEY_DATA,
     TEST_OPENSSH_KEY_DATA_LOCKED,
     TEST_SSH_CERT_KEY,
+    TEST_CATTED_SSH_KEY_DATA,
 )
 from rest_framework.serializers import ValidationError as RestValidationError
 
 import pytest
 
 
+def test_invalid_keys():
+    invalid_keys = [
```

Comment:
I introduced a number of negative tests I know should fail based on your comments.